### ISSUE TITLE
Low performance using this library

### ISSUE THREAD

**Issue by ohadkorenok** [mybigday/whisper.rn#263]
Hey, i'm trying to run ggml-base-q5_1.bin on my mobile and its still very slow. Do you know how can it be improved?

**Comment by sirramin** [comment]
You can use onnx as an alternative

**Comment by ohadkorenok** [comment]
I don't think it's related to the model as much to the memory leak in the code....
Do you have a realtime example that works for you?

**Comment by sirramin** [comment]
https://github.com/microsoft/onnxruntime-inference-examples/tree/main/mobile/examples/React_Native/ort-rn-basic-usage

**Comment by ohadkorenok** [comment]
looking good bro but have you tried that? it has good results?

**Comment by sirramin** [comment]
Yes I implemented it in my app. It has   much better performance and better
accuracy.

**Comment by Martin000N** [comment]
Where exactly does the memory leak happen?

**Comment by ohadkorenok** [comment]
In realtime processing. I can't use it for too much long
